### DISCUSSION TITLE
slider velocity not saving

### DISCUSSION THREAD

**Discussion by Axilotl17** [ppy/osu#17257] [Q&A] [answered]
this is a bit complex, so ive just attached a video detailing the bug. i ramble a bit at the end so you can skip that. 

video:
https://youtu.be/MtQ0U5f720Q
video TLDR: the map i mostly made in lazer, uploaded and slightly edited in stable, now doesn't work in lazer but does work in stable
(doesnt work meaning slider velocities arent saved right)

map: 
https://osu.ppy.sh/beatmapsets/1719878#osu/3514788

osu lazer map export (its a little different from uploaded version, but the first 2x velocity slider is the same):
[BLUE ENCOUNT - Survivor (Axilotl).zip](https://github.com/ppy/osu/files/8250078/BLUE.ENCOUNT.-.Survivor.Axilotl.zip)

standard osu map export:
[BLUE ENCOUNT - Survivor.zip](https://github.com/ppy/osu/files/8250085/BLUE.ENCOUNT.-.Survivor.zip)

i dont think logs help here

**Comment by frenzibyte** [comment, marked as answer]
Escalated to #17258
